src/app/(app)/Components/Carousel.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+
+import React from "react";
+import dynamic from "next/dynamic";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import {BiSolidLeftArrow, BiSolidRightArrow} from "react-icons/bi";
+
+const Slider = dynamic(() => import("react-slick"), {ssr: false});
+
+const NextArrow = (props: { onClick?: () => void }) => {
+    const {onClick} = props;
+    return (
+        <div
+            style={{
+                position: "absolute",
+                right: 0,
+                top: "50%",
+                transform: "translateY(-50%)",
+                zIndex: 1,
+                background: "linear-gradient(to left, rgba(0, 0, 0, 0.95) 25%, transparent 100%)",
+                height: "100%",
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "center",
+                cursor: "pointer",
+                width: "100px",
+            }}
```

Comment:
Let's stick to tailwind-css for consistency

Reply:
I replaced the css with tailwind classes except for the background.